scripts/deployment/phase2-assets/collaterals/deploy_morpho_aavev2_plugin.ts

```diff
@@ -0,0 +1,150 @@
+    throw new Error(`Missing network configuration for ${hre.network.name}`)
+  }
+
+  // Get phase1 deployment
+  const phase1File = getDeploymentFilename(chainId)
+  if (!fileExists(phase1File)) {
+    throw new Error(`${phase1File} doesn't exist yet. Run phase 1`)
+  }
+  // Check previous step completed
+  const assetCollDeploymentFilename = getAssetCollDeploymentFilename(chainId)
+  const assetCollDeployments = <IAssetCollDeployments>getDeploymentFile(assetCollDeploymentFilename)
+
+  const deployedCollateral: string[] = []
+  const revenueHiding = fp('1e-6').toString() // revenueHiding = 0.0001%
+
+  /******** Deploy Morpho - AaveV2 **************************/
+
+  /******** Morpho token vaults **************************/
+  console.log(`Deploying morpho token vaults to network ${hre.network.name} (${chainId})
+    with burner account: ${deployer.address}`)
+  const MorphoTokenisedDepositFactory = await ethers.getContractFactory("MorphoAaveV2TokenisedDeposit")
+  const maUSDT = await MorphoTokenisedDepositFactory.deploy({
+    morphoController: networkConfig[chainId].MORPHO_AAVE_CONTROLLER!,
+    morphoLens: networkConfig[chainId].MORPHO_AAVE_LENS!,
+    rewardsDistributor: networkConfig[chainId].MORPHO_REWARDS_DISTRIBUTOR!,
+    underlyingERC20: networkConfig[chainId].tokens.USDT!,
+    poolToken: networkConfig[chainId].tokens.aUSDT!,
+    rewardToken: networkConfig[chainId].tokens.MORPHO!,
+  })
+
+  const maUSDC = await MorphoTokenisedDepositFactory.deploy({
+    morphoController: networkConfig[chainId].MORPHO_AAVE_CONTROLLER!,
+    morphoLens: networkConfig[chainId].MORPHO_AAVE_LENS!,
+    rewardsDistributor: networkConfig[chainId].MORPHO_REWARDS_DISTRIBUTOR!,
+    underlyingERC20: networkConfig[chainId].tokens.USDC!,
+    poolToken: networkConfig[chainId].tokens.aUSDC!,
+    rewardToken: networkConfig[chainId].tokens.MORPHO!,
+  })
+  const maDAI = await MorphoTokenisedDepositFactory.deploy({
+    morphoController: networkConfig[chainId].MORPHO_AAVE_CONTROLLER!,
+    morphoLens: networkConfig[chainId].MORPHO_AAVE_LENS!,
+    rewardsDistributor: networkConfig[chainId].MORPHO_REWARDS_DISTRIBUTOR!,
+    underlyingERC20: networkConfig[chainId].tokens.DAI!,
+    poolToken: networkConfig[chainId].tokens.aDAI!,
+    rewardToken: networkConfig[chainId].tokens.MORPHO!,
+  })
+
+  await maUSDT.deployed()
+  await maUSDC.deployed()
+  await maDAI.deployed()
+
+  assetCollDeployments.erc20s.maUSDT = maUSDT.address
+  assetCollDeployments.erc20s.maUSDC = maUSDC.address
+  assetCollDeployments.erc20s.maDAI = maDAI.address
+
+  /******** Morpho collateral **************************/
+  const FiatCollateralFactory = await hre.ethers.getContractFactory(
+    "MorphoFiatCollateral"
+  )
+
+  {
+    const collateral = await FiatCollateralFactory.connect(deployer).deploy({
+      priceTimeout: priceTimeout.toString(),
+      chainlinkFeed: networkConfig[chainId].chainlinkFeeds.USDT!,
+      oracleError: fp('0.005').toString(), // 0.5%
```

Comment:
it's 0.25% over [here](https://docs.chain.link/data-feeds/price-feeds/addresses)